crates/cairo-coverage/src/main.rs

```diff
@@ -12,6 +16,10 @@ use std::io::Write;
 fn main() -> Result<()> {
     let cli = Cli::parse();
 
+    let input_data = InputData::try_from(&cli.trace_file)?;
+    let intermediate_representation = FunctionsByFile::from(&input_data);
```

Comment:
```suggestion
    let coverage_data = FunctionsByFile::from(&input_data);
```
intermediate representation is a weird name here. We do not operate on any AST.